pkg/openthread/patches/0001-fix-naming-conflict-mkversion.patch

```diff
@@ -0,0 +1,26 @@
+From 829cfafe163fe6557446f62834b93b55266c364f Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Jos=C3=A9=20Ignacio=20Alamos=20Aste?= <jialamos@uc.cl>
+Date: Wed, 15 Jun 2016 16:41:00 +0200
+Subject: [PATCH] Added patch
+
+---
+ third_party/nlbuild-autotools/repo/scripts/mkversion | 3 ++-
+ 1 file changed, 2 insertions(+), 1 deletion(-)
+
+diff --git a/third_party/nlbuild-autotools/repo/scripts/mkversion b/third_party/nlbuild-autotools/repo/scripts/mkversion
+index aa0b9f8..eaddc22 100755
+--- a/third_party/nlbuild-autotools/repo/scripts/mkversion
++++ b/third_party/nlbuild-autotools/repo/scripts/mkversion
+@@ -36,7 +36,8 @@
+ 
+ # Constants
+ 
+-ROOTDIR=${PREFIX}/
++#ROOTDIR=${PREFIX}/
++ROOTDIR=/
```

Comment:
There's a conflict between mkversion's PREFIX and RIOT PREFIX variable. This is not the cleanest solution, but for now it's working.